test/errors.js

```diff
@@ -1106,6 +1106,31 @@ ${M}      </pre>
       );
     });
 
+    it('unknown oneof', async () => {
+      await assertError(
+        positioned`
+        <emu-grammar type="definition">
+          Foo :: one of \`a\` \`b\` \`c\`
+        </emu-grammar>
+
+        <emu-clause id="sec-example" type="sdo">
+        <h1>Static Semantics: Example</h1>
+        <dl class='header'></dl>
+          <emu-grammar>
+            Foo :: ${M}one of \`a\`
+          </emu-grammar>
+          <emu-alg>
+            1. Return *true*.
+          </emu-alg>
+        </emu-clause>`,
+        {
+          ruleId: 'grammar-shape',
+          nodeType: 'emu-grammar',
+          message: 'could not find definition for rhs "a"',
```

Comment:
Why is this reporting an error, if `a` is one of the valid values?

Reply:
Apologies, it's late here. I guess what we actually want is a subset check?

Reply:
Probably, yes 👍

Reply:
Done, also added a negative test to cover this.